staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1261,7 +1278,8 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
 		}
 	}
 
-	glogBody("Response Body", body)
+	// Call depth is tricky. This one is okay for Do and DoRaw.
+	logBody(ctx, 7, "Response Body", body)
```

Comment:
these call depths are brittle, aren't they? How can we make sure the output is not nonsense quickly?

Reply:
There's a test which checks that the result of stack unwinding is the source code of the caller of client-go. If it goes wrong, we end up in client-go code (which is what we do now) or go up too far in the app on top of client-go (which is still at least somewhat relevant).

I think that's better than always logging `request.go` as source code location. Most users (probably?) won't care about the code in client-go which emits the log message and more about their own code which does the request.

The one downside is if some developer wants to find the log call (they can search for the string) or if someone uses source+line as ID for the log call (they shouldn't!).